webpack.config.js

```diff
@@ -97,10 +105,15 @@ module.exports = {
             {from: 'intl', to: 'js'}
         ]),
         new webpack.optimize.UglifyJsPlugin({
+            mangle: true,
             compress: {
                 warnings: false
             }
         }),
+        new webpack.DefinePlugin({
+            'process.env.NODE_ENV': '"' + process.env.NODE_ENV + '"'
```

Comment:
Switches Redux and React in/out of production mode.  Redux reports a warning if you run it minified but with NODE_ENV !== "production", which is expected since we run minified code in development.